includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -807,6 +810,9 @@ public function payment_fields() {
 
 			</fieldset>
 			<?php
+
+			do_action( 'wc_payment_fields_wcpay_end', $this->id );
```

Comment:
That's a good point. 

The reason for adding both, the `*_start` and `*_end` filters, was due to a misinterpretation of the following:
> In some cases, the merchant might want to further customise their checkout and add something before or after the card fields.

From issue #3230.

But upon re-reading this statement, I can see how it can be interpreted as being on or the other but not both :)